aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/SRPHelper.kt

```diff
@@ -16,7 +16,7 @@
 package com.amplifyframework.auth.cognito.helpers
 
 import androidx.annotation.VisibleForTesting
-import com.amplifyframework.auth.AuthException
+import com.amplifyframework.auth.cognito.helpers.AuthHelper.Companion.HMAC_SHA_256
```

Comment:
We can define this constant in SRPHelper can remove this import.

Reply:
It is required in both classes. Are you suggesting we duplicate it?

Reply:
Yes